package.json

```diff
@@ -31,6 +31,8 @@
     "templates/"
   ],
   "dependencies": {
+    "chalk": "^1.1.1",
```

Comment:
Or, let's use `chalk` everywhere (in a follow-up), since in most "clarifications" we're removing gutil.log.